app/packs/src/components/searchModal/forms/AdvancedSearchRow.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {Button, FormControl} from 'react-bootstrap'
+import Select from 'react-select';
+
+import SelectFieldData from './SelectFieldData';
+import SelectMapperData from './SelectMapperData';
+
+const AdvancedSearchRow = ({ idx, selection, onChange }) => {
+  const mapperOptions = SelectMapperData.fields;
+  const fieldOptions = SelectFieldData.fields;
+  const andOrOptions = [
+    { value: "AND", label: "AND" },
+    { value: "OR", label: "OR" }
+  ];
+
+  const combinationSelect = () => {
```

Comment:
Frage: ich überleg die ganze Zeit wie man inner ersten Zeile den Logischen Operator wegmachen kann... wärs möglich den zwar mitzuführen aber einfach per CSS auszublenden? so per first-child oder first-of(class) direktiven? dann spart man sich n paar sonderfälle und hat einheitliche Daten